### PR TITLE
Leaderboard: Final Touches

### PR DESCRIPTION
**Pull request by Shisui6**
## Design Application Look and Feel
- Add styling to application
- Implement loading skeletons 
- Add animations to DOM elements

### CODE REVIEW

File: src/modules/leaderboard.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+import profile3 from '../images/profile3.png';
+import profile4 from '../images/profile4.png';
+
 // Get relevant elements from the DOM
 const scores = document.getElementById('scores');
+const loader = document.getElementById('skeleton-loader');
+const empty = document.getElementById('empty-id');
 
 // URL to be used in fetch requests
 const URL = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${process.env.API_KEY}/scores/`;
```
Comment (franclobo) [inline src/modules/leaderboard.js:12]: - [ ] **[OPTIONAL]** You are using all the data charged on the API and it is difficult to see my score on the list, please 🙏🏽  assign a valid name to the game for a better user experience. 👇🏽 

**HINT**
Check this [tutorial](https://www.loom.com/share/4f22d42eaec44bc88ddf12548acb9a57)

File: src/style.css [hunk 2]
```diff
@@ -1,72 +1,248 @@
...
+}
+
+.leaderboard {
+  box-shadow: 0 4px 8px 0 rgba(255, 255, 255, 0.2);
+  border-radius: 35px;
+  padding: 20px;
+  height: 370px;
+  background-color: #fff;
+  overflow-y: auto;
+}
```
Comment (franclobo) [inline src/style.css:97]: - [ ] The width of the leaderboard is too big, please 🙏🏽 fix it for a better user interface the layout should look like the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png).

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Shisui6, 

# Required Changes ♻️

Good job so far!
You are doing it good 👍🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly review the suggestion to complete the activity.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — COMMENTED** [review]
Hello, @Shisui6👋,

_This is an additional review due to the second opinion request._

It is true that if the APP is setup as described in the README file it runs as expected and as evident in the screenshot below:

| image |
| --- |

But, just in case someone needed to take a look at your app without adding a game id, the behavior that @franclobo found above becomes true. Please consider working with some error boundaries so that your UI remains the same in all situations. Lack of state should not break the UI.

@franclobo, please always take a look at the README file when setting up any project for review.

Please DON'T ask for another review in your Dashboard. As this was the only reason your project was not approved it will be approved and you'll be able to submit this task on your dashboard ASAP.

Keep rocking 🚀!

**Comment by Shisui6** [comment]
Hey @Lameck1,

Thanks for your review. The project is still being shown as `request review` in my dashboard so I am unable to submit it. Please advise.

image

**Comment by Shisui6** [comment]
Hey @Lameck1,

Thanks for your review. The project is still being shown as `request review` in my dashboard so I am unable to submit it. Please advise.

image

**Pull request merged**